lib/route/src/Obelisk/Route.hs

```diff
@@ -816,14 +823,20 @@ _R
   -> Prism' (R f) a
 _R variant = dSumGEqPrism variant . iso runIdentity Identity
 
+{-# DEPRECATED prismEncoder "Use reviewEncoder instead" #-}
 -- | An encoder that only works on the items available via the prism. An error will be thrown in the parse monad
 -- if the prism doesn't match.
 prismEncoder :: (Applicative check, MonadError Text parse) => Prism' b a -> Encoder check parse a b
-prismEncoder p = unsafeMkEncoder $ EncoderImpl
-  { _encoderImpl_encode = (^. re p)
+prismEncoder = reviewEncoder
+
+-- | An encoder that only works on the items available via the prism. An error will be thrown in the parse monad
+-- if the prism doesn't match.
+reviewEncoder :: (Applicative check, MonadError Text parse) => Prism' b a -> Encoder check parse a b
```

Comment:
Let's call out that this is contravariant just like `review` from lens. Otherwise everything looks great.

Reply:
Okie, added documentation, is that good?

Reply:
Whoops just forgot the `f` at the end in both cases.

Reply:
eek, thank you!